packages/contracts-bedrock/src/universal/StorageSetter.sol

```diff
@@ -0,0 +1,46 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.15;
+
+import { ISemver } from "src/universal/ISemver.sol";
+import { Storage } from "src/libraries/Storage.sol";
+
+/// @title StorageSetter
+/// @notice A simple contract that allows setting arbitrary storage slots.
+///         WARNING: this contract is not safe to be called by untrusted parties.
+///         It is only meant as an intermediate step during upgrades.
+contract StorageSetter is ISemver {
+    /// @notice Semantic version.
+    /// @custom:semver 1.0.0
+    string public constant version = "1.0.0";
```

Comment:
Should this be kept to < 0 major so as to add another indicator that this shouldn't be used outside the upgrade workflow (non-prod environments)?

Reply:
I see what you mean, but this is being used in the prod upgrade flow